doc/tutorials/installation.rst

```diff
@@ -156,7 +156,7 @@ RLE images provided a suitable plugin is installed.
 
 Using pip::
 
-  pip install pylibjpeg pylibjpeg-libjpeg pylibjpeg-openjpeg pylibjpeg-rle
+  pip install "pylibjpeg>=1.2" pylibjpeg-libjpeg pylibjpeg-openjpeg pylibjpeg-rle
```

Comment:
The pip install command should automatically install the newest version, so why is this necessary?

Reply:
to do you would have to have there `--upgrade` argument
also with your assumptions if you have it installed some time ago or other library restrict version you won't find it as an install pass but it is not feasible version

Reply:
so the options are:
- `pip install "pylibjpeg>=1.2" pylibjpeg-libjpeg pylibjpeg-openjpeg pylibjpeg-rle`
- `pip install -U pylibjpeg pylibjpeg-libjpeg pylibjpeg-openjpeg pylibjpeg-rle`

Reply:
Fair enough, I'd rather the `-U` version though

Reply:
```suggestion
  pip install -U "pylibjpeg>=1.2" pylibjpeg-libjpeg pylibjpeg-openjpeg pylibjpeg-rle
```